dotnet/Data/WishListRepository.cs

```diff
@@ -229,12 +240,13 @@ public async Task VerifySchema()
             }
         }
 
-        public async Task<WishListsWrapper> GetAllLists()
+        private async Task<string> firstScroll()
```

Comment:
Please use Pascal Case: FirstScroll

Reply:
Hey Brian, thank you for the comments. 
I tried to run the first scroll call and all the sub calls in the postman and found that they all have the X-VTEX-MD-TOKEN in the header. The first sub call always has the same token as the initial call, and other sub calls' tokens can be updated by themselves sometimes.
So I think checking if the token is present is not a good idea since we want to make sure all sub calls use the token from the first scroll call. 

I will update functions to Pascal Case.